package.json

```diff
@@ -1,7 +1,7 @@
 {
   "name": "react-native-pdftron",
   "title": "React Native Pdftron",
-  "version": "3.0.2-beta.12",
+  "version": "3.0.2-beta.11",
```

Comment:
Should be 13?

Reply:
Thanks for pointing that out, version number has been fixed